src/fabric/src/fabric_doc_update.erl

```diff
@@ -106,11 +106,11 @@ before_doc_update(DbName, Docs, Opts) ->
         {true, _} ->
             %% fake db is expensive to create so we only do it if we have to
             Db = fabric_util:fake_db(DbName, Opts),
-            [couch_replicator_manager:before_doc_update(Doc, Db) || Doc <- Docs];
+            [couch_replicator_manager:before_doc_update(Doc, Db, []) || Doc <- Docs];
```

Comment:
According to update description of the PR the update type can be either:
- replicated_changes
- interactive_updates